config/locales/en.yml

```diff
@@ -24,19 +24,18 @@ en:
     title: Cookies
     banner:
       title: "Can we store analytics cookies on your device?"
-      text: "Analytics cookies help us understand how our website is being used."
+      text: "Analytics cookies help us understand how our service is being used."
       confirmation_message: "You’ve accepted all cookies. You can <a class='govuk-link' href='/cookies'>change your cookie settings</a> at any time."
```

Comment:
Just a small thing, but for consistency this should probably read "You’ve accepted analytics cookies..."